Client/game_sa/CModelInfoSA.cpp

```diff
@@ -1728,6 +1734,69 @@ void CModelInfoSA::MakeVehicleAutomobile(ushort usBaseID)
     CopyStreamingInfoFromModel(usBaseID);
 }
 
+bool GetFirstAtomic(RpAtomic* atomic, void* data)
+{
+    RpAtomic** firstAtomicPtr = (RpAtomic**)data;
+
+    if (*firstAtomicPtr == nullptr)
+        *firstAtomicPtr = atomic;
+
+    return false;
+}
+
+bool CModelInfoSA::MakeAtomicModel(ushort usModelId)
```

Comment:
What is the purpose of usModelId? Why don't just use m_dwModelID instead? Or usModelId can differ?